src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -134,10 +134,9 @@ std::optional<T> get_option(ov::AnyMap& config, const std::string& option_name)
 }
 
 template <typename T>
-T pop_or_default(ov::AnyMap& config, const std::string& key, const T& default_value) {
-    auto anyopt = pop_option(config, key);
-    if (anyopt.has_value()) {
-        return anyopt.value().as<T>();
+T opt_or_default(const std::optional<ov::Any>& opt, const T& default_value) {
```

Comment:
Nitpick: Well, it's not really `opt` or `default` in this sense.

It's `value_or_default` and this functionality already exists in `std::optional`: https://en.cppreference.com/w/cpp/utility/optional/value_or

Reply:
Btw, what is wrong with `pop_or_default`?

Reply:
Removed